py_irt/models/two_param_logistic.py

```diff
@@ -271,6 +273,17 @@ def fit_MCMC(self, models, items, responses, num_epochs):
         print(theta_sum)
         print(b_sum)
 
+    def predict(self, models, items, params_from_file=None):
```

Comment:
Ditto on subjects and doc comment being triple quote

Reply:
Updated throughout.